app/src/org/commcare/dalvik/services/CommCareSessionService.java

```diff
@@ -215,7 +212,7 @@ private void showLoggedOutNotification() {
         // The PendingIntent to launch our activity if the user selects this notification
         Intent i = new Intent(this, CommCareHomeActivity.class);
         
-        PendingIntent contentIntent = PendingIntent.getActivity(this, 0, i, PendingIntent.FLAG_ONE_SHOT);
+        PendingIntent contentIntent = PendingIntent.getActivity(this, 0, i, PendingIntent.FLAG_CANCEL_CURRENT);

```

Comment:
This shouldn't be FLAG_ONE_SHOT because you may want to click the "Session expired" pinned notification multiple times and have it launch the login activity. I think FLAG_UPDATE_CURRENT works https://developer.android.com/reference/android/app/PendingIntent.html#FLAG_UPDATE_CURRENT